### PR TITLE
first try on cross entropy

### LINKED ISSUE #2: Cross entropy loss function

### LINKED ISSUE #7: Binary cross entropy loss function

### PR DESCRIPTION
**Pull request by ninjalu**
First try
fixes #7

### CODE REVIEW

**Review by life-efficient — APPROVED** [review]
Best code i ever saw
